apis/database/v1beta1/cloudsql_instance_types.go

```diff
@@ -349,7 +349,7 @@ type IPConfiguration struct {
 
 	// PrivateNetwork: The resource link for the VPC network from which the
 	// Cloud SQL instance is accessible for private IP. For example,
-	// /projects/myProject/global/networks/default. This setting can be updated,
```

Comment:
This is indeed the same comment as the one gcloud API: https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1beta4/instances#IpConfiguration

However, using this format (with leading /) does not work actually:

```
create failed: cannot create new CloudSQL instance: googleapi: Error 400: Invalid request: Project redacted-project has invalid private network name /projects/redacted-project/global/networks/default., invalid
```